api/package.json

```diff
@@ -26,7 +26,7 @@
     "dev": "nodemon ./src/server.ts",
     "dev:debug": "nodemon --inspect ./src/server.ts",
     "docs": "npm run docs:external && npm run docs:internal",
-    "docs:external": "mkdir -p local/docs/external; swagger-inline src/routes/*.ts src/types/commontypes.ts --base docs/swagger.base.yaml --format yaml > local/docs/external/swagger.yaml; redoc-cli build local/docs/external/swagger.yaml --options.hideDownloadButton=true --output=local/docs/external/index.html --title=\"Touca Server API Reference\"",
+    "docs:external": "mkdir -p local/docs/external; swagger-inline src/routes/*.ts ../packages/api-schema/src/*.ts --base docs/swagger.base.yaml --format yaml > local/docs/external/swagger.yaml; redoc-cli build local/docs/external/swagger.yaml --options.hideDownloadButton=true --output=local/docs/external/index.html --title=\"Touca Server API Reference\"",
```

Comment:
We agreed that this is a temporary workaround and that future improvements will help us fix this.